lighthouse-core/audits/audit.js

```diff
@@ -72,6 +72,7 @@ class Audit {
    * considering a log-normal distribution governed by the two control points, point of diminishing
    * returns and the median value, and returning the percentage of sites that have higher value.
    *
+   * NOTE: deprecated. Prefer `computeLogNormalScoreFrom90th()`.
```

Comment:
not sure what to do with this. Fine to drop for us after this PR, but at least `publisher-ads-lighthouse-plugin` uses this method ([example](https://github.com/googleads/publisher-ads-lighthouse-plugin/blob/764da40258c73cd8a2f1b29a9126a01a51ed71bf/lighthouse-plugin-publisher-ads/audits/ad-request-from-tag-load.js#L82)), so this would be a pretty breaking change

Reply:
yah this seems good.